lib/pages/chat/events/download_video_widget.dart

```diff
@@ -180,6 +170,16 @@ class _DownloadVideoWidgetState extends State<DownloadVideoWidget>
               ),
             ],
           ),
+          TwakeIconButton(
+            margin: VideoViewerStyle.backButtonMargin(context),
+            tooltip: L10n.of(context)!.back,
+            icon: Icons.close,
+            onTap: () {
+              cancelToken.cancel();
+              Navigator.of(context).pop();
+            },
+            iconColor: Theme.of(context).colorScheme.surface,
+          ),
```

Comment:
what is the different with your change?

Reply:
I make it to the last item in the stack, so that it can be clickable